devicetypes/sensative/sensative-strips-guard-700.src/sensative-strips-guard-700.groovy

```diff
@@ -0,0 +1,482 @@
+	
+	if (!device.currentValue("tamper")) {
+		sendEventIfNew("tamper", "clear")
+	}
+	
+	if (!device.currentValue("checkInterval")) {
+		sendEvent(name: "checkInterval", value: ((wakeUpIntervalSeconds * 2) + 300), displayed: falsle, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	}
+}
+
+
+def configure() {
+	logDebug "configure()..."
+
+	state.pendingRefresh = true
+	sendCommands(getConfigureCmds())
+}
+
+List<String> getConfigureCmds() {
+	runIn(6, refreshSyncStatus)
+
+	int changes = pendingChanges
+	if (changes) {
+		log.warn "Syncing ${changes} Change(s)"
+	}
+
+	List<String> cmds = [ ]
+
+	if (state.pendingRefresh) {		
+		cmds << batteryGetCmd()
+		cmds << versionGetCmd()
+		cmds << sensorBinaryGetCmd()
+	}
+	
+	if (state.pendingRefresh || (state.wakeUpInterval != wakeUpIntervalSeconds)) {
+		logDebug "Changing wake up interval to ${wakeUpIntervalSeconds} seconds"
+		cmds << wakeUpIntervalSetCmd(wakeUpIntervalSeconds)
+		cmds << wakeUpIntervalGetCmd()
+	}
+
+	configParams.each {
+		Integer storedVal = getParamStoredValue(it.num)
+		Integer settingVal = getSettingValue(it.num)
+		
+		if ((settingVal != null) && (settingVal != storedVal)) {
+			logDebug "CHANGING ${it.name}(#${it.num}) from ${storedVal} to ${settingVal}"
+			cmds << configSetCmd(it, settingVal)
+			cmds << configGetCmd(it)
+		}
+		else if (state.pendingRefresh) {
+			cmds << configGetCmd(it)
+		}
+	}
+
+	state.pendingRefresh = false
+	return cmds
+}
+
+
+// Required for HealthCheck Capability, but doesn't actually do anything because this device sleeps.
+def ping() {
+	logDebug "ping()"
+}
+
+
+def refresh() {
+	logDebug "refresh()..."
+
+	state.pendingRefresh = true
+	logForceWakeupMessage("The device will be refreshed the next time it wakes up.")	
+}
+
+void logForceWakeupMessage(String msg) {
+	log.warn "${msg}  To force the device to wake up immediately, move the magnet towards the round end 3 times."
+}
+
+
+String wakeUpIntervalGetCmd() {
+	return secureCmd(zwave.wakeUpV2.wakeUpIntervalGet())
+}
+
+String wakeUpIntervalSetCmd(seconds) {
+	return secureCmd(zwave.wakeUpV2.wakeUpIntervalSet(seconds:seconds, nodeid:zwaveHubNodeId))
+}
+
+String wakeUpNoMoreInfoCmd() {
+	return secureCmd(zwave.wakeUpV2.wakeUpNoMoreInformation())
+}
+
+String batteryGetCmd() {
+	return secureCmd(zwave.batteryV1.batteryGet())
+}
+
+String versionGetCmd() {
+	return secureCmd(zwave.versionV1.versionGet())
+}
+
+String sensorBinaryGetCmd() {
+	return secureCmd(zwave.sensorBinaryV1.sensorBinaryGet())
+}
+
+String configSetCmd(Map param, int value) {
+	return secureCmd(zwave.configurationV2.configurationSet(parameterNumber: param.num, size: param.size, scaledConfigurationValue: value))
+}
+
+String configGetCmd(Map param) {
+	return secureCmd(zwave.configurationV2.configurationGet(parameterNumber: param.num))
+}
+
+String secureCmd(cmd) {
+	try {
+		if (zwaveInfo?.zw?.contains("s") || ("0x98" in device?.rawDescription?.split(" "))) {
```

Comment:
Please, could You use a constant here instead of 0x98 ?

Reply:
New devices don't require the 0x98 check for the old style fingerprints so it will be removed from the new version instead of changed into a constant.